### ISSUE TITLE
yarn install for v1.3.0 does not work

### ISSUE THREAD

**Issue by akharris** [dfreeman/ember-ace#23]
When I tried bumping the version from `1.2.0` to `1.3.0` the package that was installed was missing many directories (including `addon/component`). It seems like there's something wrong with the package in the yarn registry for `1.3.0`. I'm new to yarn so it's possible that I'm overlooking something. 

I'm going to keep looking into this issue.

from the git diff

```
 ember-ace@^1.2.0:
-  version "1.2.0"
-  resolved "https://registry.yarnpkg.com/ember-ace/-/ember-ace-1.2.0.tgz#93115b20f99c18c49ccaea026fcbf8c00c175997"
+  version "1.3.0"
+  resolved "https://registry.yarnpkg.com/ember-ace/-/ember-ace-1.3.0.tgz#bc39256d2ed853a92f7bba7c96ccbc30a9f364ee"
```

tree of ember-ace 1.2.0 from yarn registry
```
➜  ember-ace-1-2-0 tree .
.
├── LICENSE.md
├── README.md
├── addon
│   ├── components
│   │   ├── ember-ace-completion-tooltip.js
│   │   └── ember-ace.js
│   ├── index.js
│   ├── templates
│   │   └── components
│   │       ├── ember-ace-completion-tooltip.hbs
│   │       └── ember-ace.hbs
│   └── utils
│       └── completion-manager.js
├── addon-test-support
│   ├── components
│   │   ├── ember-ace
│   │   │   ├── annotation.js
│   │   │   ├── autocomplete.js
│   │   │   ├── line.js
│   │   │   ├── marker.js
│   │   │   └── token.js
│   │   └── ember-ace.js
│   └── helpers
│       ├── editor-interaction.js
│       └── poll-condition.js
├── app
│   └── components
│       ├── ember-ace-completion-tooltip.js
│       └── ember-ace.js
├── config
│   └── environment.js
├── index.js
├── lib
│   └── worker-manifest.js
├── package.json
├── test-support
│   └── components
│       └── ember-ace.js
└── yarn.lock

15 directories, 24 files
```

tree view of ember 1.3.0 from yarn registry

```
➜  ember-ace-1-3-0 tree .
.
├── LICENSE.md
├── README.md
├── addon
│   ├── index.js
│   ├── templates
│   └── utils
│       └── completion-manager.js
├── addon-test-support
│   └── helpers
│       ├── editor-interaction.js
│       └── poll-condition.js
├── app
├── config
│   └── environment.js
├── index.js
├── lib
│   └── worker-manifest.js
├── package.json
└── test-support

9 directories, 10 files
```

**Comment by dfreeman** [comment]
Sorry about that @akharris! It looks like something went wrong with the publish. I've published 1.3.1 (which looks to have all the files) and deprecated 1.3.0 on the registry. Can you give 1.3.1 a try?

**Comment by akharris** [comment]
1.3.1 works as expected. Thanks!
